src/App.tsx

```diff
@@ -3,6 +3,8 @@ import DashboardScreen from "./components/screens/dashboard/dashboard-screen";
 import ErrorScreen from "./components/screens/error-screen";
 import AuthenticationProvider from "./components/providers/authentication-provider";
 import ErrorHandler from "./components/contexts/error-handler";
+import { ThemeProvider } from "@emotion/react";
```

Comment:
This looks like a different import from what is used in OSS, is that deliberate? I'm not familiar with the details for setting up a theme, so if it works and you're happy with it then so am I!

Reply:
According to the docs, createTheme is imported from "@mui/material". Thanks for pointing that out!